taskt/Core/Automation/Commands/Variable Commands/AddToVariableCommand.cs

```diff
@@ -1,63 +1,63 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
-using System.Data;
 using System.Collections.Generic;
 using System.Windows.Forms;
-using System.Drawing;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Variable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to modify variables.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to modify the value of variables.  You can even use variables to modify other variables.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements actions against VariableList from the scripting engine.")]
+    [Group("Variable Commands")]
+    [Description("This command adds an item to a list variable of string type.")]
     public class AddToVariableCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select a variable to modify")]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
+        [PropertyDescription("Variable Name")]
```

Comment:
This appears to be an output right? Please follow the template format here, in the rendering, and display. It should go last. Otherwise, if it's an input, remove the attributes that make it look like an output such as the remarks and the v_userVariableName. Also this command appears to be misnamed and in the wrong location. If I'm not mistaken, its function is to add an item to a list variable. Should it not go under Data Commands?

Reply:
I think we should get rid of this command as there is no need of this for now.
For adding an item to a List, we can create a command later.

Reply:
This command still needs to be moved to Data Commands

Reply:
Renamed, Updated, and Moved to the Data Commands